eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Bbody.hs

```diff
@@ -308,3 +314,16 @@ instance
   where
   wrapFailed = LedgersFailure
   wrapEvent = ShelleyInAlonzoEvent . LedgersEvent
+
+totalRefScriptSizeInBlock ::
+  (EraTx era, BabbageEraTxBody era) => ProtVer -> StrictSeq (Tx era) -> UTxO era -> Int
+totalRefScriptSizeInBlock protVer txs (UTxO utxo)
+  | pvMajor protVer <= eraProtVerHigh @ConwayEra =
```

Comment:
We need to turn this feature on in Conway, while this logic would only enable it in Dijkstra
```suggestion
  | pvMajor protVer <= natVersion @10 =
```